components/butterfly/Cargo.toml

```diff
@@ -16,6 +16,7 @@ path = "../butterfly-test"
 pkg-config = "0.3"
 
 [dependencies]
+byteorder = "1.0.0"
```

Comment:
I put it here and forgot to remove it actually, good catch! (byteorder was locked to an earlier version and I was trying to navigate it without updating every single dep)